CLAUDE.local.md

```
@@ -587,791 +510,117 @@ ls -la internal/template/embedded.go
-- [ ] 특정 언어 바이너리(gopls, pylsp, rust-analyzer 등)를 "PRIMARY"로 배치하지 않았는가?
-- [ ] 16개 지원 언어(go, python, typescript, javascript, rust, java, kotlin, csharp, ruby, php, elixir, cpp, scala, r, flutter, swift)가 동등 수준으로 나열되어 있는가?
-- [ ] 특정 언어만 "enabled: true", 다른 언어는 "planned"/"deferred"로 격하하지 않았는가?
-- [ ] "Primary/Secondary/Tertiary" 분류가 있다면, 언어가 아닌 "기능"(문법 검사, 타입 체크, 린트) 기준인가?
-- [ ] 사용자 환경 감지 로직(project_markers, extension-based)이 포함되어 있는가?
-
-### 16개 지원 언어 목록
-
-Template 파일 작성 시 반드시 16개 모두 동등 처리:
+### 16개 지원 언어 (모두 동등)
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)